back-office/design/image-settings/regenerate-thumbnails.md

```diff
@@ -0,0 +1,24 @@
+Regenerate thumbnails might be a touchy operation for the user.
+
+In order to emphasize the irreversibility of this action, a yellow warning notification should be displayed before the feature's two options:
+
+_Be careful! Depending on the options selected, former manually uploaded thumbnails might be erased and replaced by automatically generated thumbnails._<br/>
+_Also, regenerating thumbnails for all existing images can take several minutes, please be patient._, localized in Admin.Design.Notification
+
+
+## Select an image
+
+[TO BE COMPLETED]
+
+
+## Erase previous images
+
+In order to avoid the user to delete his previous work to by mistake, the switc button should be set to 'No' by default.
```

Comment:
his previous work by mistake

the `switch`

Reply:
Nice, thanks for spotting the typo!